pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -1,23 +1,149 @@
-# Pyth on Solana
+---
+description: Consume Pyth Network prices in Solana applications
+---
 
-We provide one SDK for Solana on-chain programs to use Pyth prices:
+# How to Use Real-Time Data in Solana Programs
 
-The [pyth-sdk-solana crate](https://github.com/pyth-network/pyth-sdk-rs/tree/main/pyth-sdk-solana) can be used to consume Pyth prices inside Solana programs written in Rust.
+This guide explains how to use real-time Pyth data in Solana applications.
```

Comment:
This intro link is redundant.

Reply:
Yeah, I know. However, there needs to be a text block here (you can't have a heading without a text intro paragraph below it). I could expand this out a bit to say more about what the guide covers.